test/channels.jl

```diff
@@ -390,6 +390,7 @@ end
         t = Timer(0) do t
             tc[] += 1
         end
+        Libc.systemsleep(0.005)
```

Comment:
Is there something better we can do here. Tests that depend on system scheduling behavior almost always turn out flakey.

Reply:
Without this, the test depends on the scheduling (this test assumes that this statement takes a least a millisecond to complete). This completely fixes the test to avoid that flakiness by ensuring it always takes at least that long.

Reply:
Ah, I see.